```
+ go [list -e -json -compiled=true -test=false -export=false -deps=true -find=false -tags  -- github.com/status-im/status-go/mobile]: exit status 2: # runtime/cgo
gcc_android.c:6:10: fatal error: android/log.h: No such file or directory
 #include <android/log.h>
          ^~~~~~~~~~~~~~~
compilation terminated.

Makefile:93: recipe for target 'statusgo-android' failed
make: *** [statusgo-android] Error 1
```
Seems like something is missing, but what? Maybe Android SDK?
```
 $ ansible ci-slave-linux -a 'find /usr/lib/android-sdk -name log.h'

linux-03.do-ams3.ci.misc | CHANGED | rc=0 >>


linux-05.do-ams3.ci.misc | CHANGED | rc=0 >>
/usr/lib/android-sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/android/log.h
/usr/lib/android-sdk/ndk-bundle/sources/third_party/shaderc/third_party/spirv-tools/source/opt/log.h
/usr/lib/android-sdk/ndk-bundle/sources/cxx-stl/llvm-libc++/utils/google-benchmark/src/log.h
/usr/lib/android-sdk/ndk-bundle/sysroot/usr/include/android/log.h

linux-01.do-ams3.ci.misc | CHANGED | rc=0 >>
/usr/lib/android-sdk/ndk-bundle/sysroot/usr/include/android/log.h
/usr/lib/android-sdk/ndk-bundle/sources/third_party/shaderc/third_party/spirv-tools/source/opt/log.h
/usr/lib/android-sdk/ndk-bundle/sources/cxx-stl/llvm-libc++/utils/google-benchmark/src/log.h
/usr/lib/android-sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/android/log.h

linux-04.do-ams3.ci.misc | CHANGED | rc=0 >>
/usr/lib/android-sdk/ndk-bundle/sources/cxx-stl/llvm-libc++/utils/google-benchmark/src/log.h
/usr/lib/android-sdk/ndk-bundle/sources/third_party/shaderc/third_party/spirv-tools/source/opt/log.h
/usr/lib/android-sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/android/log.h
/usr/lib/android-sdk/ndk-bundle/sysroot/usr/include/android/log.h

linux-02.do-ams3.ci.misc | CHANGED | rc=0 >>
/usr/lib/android-sdk/ndk-bundle/sources/cxx-stl/llvm-libc++/utils/google-benchmark/src/log.h
/usr/lib/android-sdk/ndk-bundle/sources/third_party/shaderc/third_party/spirv-tools/source/opt/log.h
/usr/lib/android-sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/android/log.h
/usr/lib/android-sdk/ndk-bundle/sysroot/usr/include/android/log.h
```
Interestingly enough the `android/log.h` files are missing from `linux-03`, but all builds failed in this PR failed on `linux-02`, so that makes no fucking sense.

--- a/Makefile
+++ b/Makefile
@@ -212,7 +212,8 @@ xgo-install:
 	go get -u github.com/karalabe/xgo
 
 gomobile-install: xtools-install
-	go get golang.org/x/mobile/cmd/gomobile
+	go get -d golang.org/x/mobile/cmd/gomobile
+	cd $(GOPATH)/src/golang.org/x/mobile && git checkout 30c70e3810e97d051f18b66d59ae242540c0c391 && go install ./cmd/gomobile
 
 release-install:
 	go get -u github.com/c4milo/github-release

